lib/package-detail-view.coffee

```diff
@@ -201,30 +196,21 @@ class PackageDetailView extends View
     loadTime + activateTime
 
   installUpdate: ->
```

Comment:
Sounds good. I assume nothing is calling this now. Remove any dead code you can.